### PR TITLE
Correct subject of email for update request

### PR DESCRIPTION
**Pull request by amdomanska**
patch for: https://github.com/DOAJ/doajPM/issues/2570

### CODE REVIEW

**Comment by amdomanska** [comment]
Tested with Mailtrap:

Screenshot from 2021-04-28 17-55-18

**Review by richard-jones — APPROVED** [review]

File: portality/formcontext/formcontext.py [hunk 1]
```diff
@@ -492,7 +492,11 @@ def _send_application_approved_email(self, journal_title, publisher_name, email,
 
         to = [email]
         fro = app.config.get('SYSTEM_EMAIL_FROM', 'feedback@doaj.org')
-        subject = app.config.get("SERVICE_NAME", "") + " - journal accepted"
+        if update_request:
```
Comment (Steven-Eardley) [inline portality/formcontext/formcontext.py:495]: We don't use formcontext.py any more, it will be removed - changes here aren't needed.

**Review by Steven-Eardley — CHANGES_REQUESTED** [review]
This causes a genuine test failure - we need some updates in doajtest/unit/application_processors/test_application_processor_emails.py

The first test needs updating:
* It uses an update request fixture where it should be a new application
* It doesn't check both update requests and new applications

I assume the fixture was repurposed after this test was written, since it predates update requests.

File: doajtest/unit/application_processors/test_application_processor_emails.py [hunk 2]
```diff
@@ -915,7 +915,7 @@ def test_01_maned_review_emails(self):
         #   * to the journal contact, informing them of the journal's acceptance
         publisher_template = 'publisher_update_request_accepted.txt'
         publisher_to = re.escape(owner.email)
-        publisher_subject = 'journal accepted'
+        publisher_subject = 'update request accepted'
```
Comment (amdomanska) [inline doajtest/unit/application_processors/test_application_processor_emails.py:918]: test updated

**Pull request closed without merging**